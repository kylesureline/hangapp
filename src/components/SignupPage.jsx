import React from 'react';
import Signup from './Signup.jsx';

const SignupPage = () => (
  <div className="box-layout">
    <div className="box-layout__box">
      <h1 className="box-layout__title">Signup</h1>
      <Signup />
    </div>
  </div>
);

export default SignupPage;
