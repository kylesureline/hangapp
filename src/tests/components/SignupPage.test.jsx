import React from 'react';
import { shallow } from 'enzyme';
import SignupPage from '../../components/SignupPage.jsx';

test('should render SignupPage correctly', () => {
  const wrapper = shallow(<SignupPage />);
  expect(wrapper).toMatchSnapshot();
});
