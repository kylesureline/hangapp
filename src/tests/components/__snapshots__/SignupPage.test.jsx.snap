// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignupPage correctly 1`] = `
<div
  className="box-layout"
>
  <div
    className="box-layout__box"
  >
    <h1
      className="box-layout__title"
    >
      Signup
    </h1>
    <Connect(Signup) />
  </div>
</div>
`;
